markup.go

```diff
@@ -47,36 +47,32 @@ type Event struct {
 
 // MarkupOrComponentOrHTML represents one of:
 //
-//  Markup
+//  MarkupList
 //  Component
 //  *HTML
+//  ChildList
 //
 // If the underlying value is not one of these types, the code handling the
 // value is expected to panic.
 type MarkupOrComponentOrHTML interface{}
```

Comment:
The name now does not reflect the actual type. But of course `MarkupListOrComponentOrHTMLOrChildListOrNil` is not a great name.

Perhaps shorter more abstract name would do us good, something like `MarkupOrChild`.

Reply:
Sounds fair to me.